src/components/gltf-model.js

```diff
@@ -22,7 +22,7 @@ module.exports.Component = registerComponent('gltf-model', {
     this.remove();
 
     this.loader.load(src, function gltfLoaded (gltfModel) {
-      self.model = gltfModel.scene;
+      self.model = gltfModel.scene || gltfModel.scenes[0];
```

Comment:
You mean, it'd be in `THREE.GLTFLoader`?
That's a good idea.
I'll talk to Three.js folks.

But at least with Three.js r84, we need this out of `THREE.GLTFLoader`.